### PR TITLE
feat: create designed home page and general fixes

### PR DESCRIPTION
**Pull request by assebc**
# Description

image
image
image
image

## Issue

https://github.com/b1tsway/b1tsway.github.io/issues/43

## Type of change

Please delete options that are not relevant or select with a "x" the correct option.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist(mark with a "x" the items you made):

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I tested all related functionalities and I didn´t break anything

### CODE REVIEW

File: src/pages/Home/styles.css [hunk 1]
```diff
@@ -12,23 +39,15 @@
...
     .home-wrapper h3{
         text-align: left;
     }
+}
+
+@media (max-width: 1600px){
+    .welcome-section li {
+        flex-direction: column;
+        align-items: flex-end;
+    }
```
Comment (Mariii-23) [inline src/pages/Home/styles.css:52]: Here we could do it like this:

```typescript
@media (max-width: 850px) {
    .home-wrapper h3{
        text-align: left;
    }

    .home-wrapper h1{
        text-align: center;
    }
}

@media (max-width: 1600px){
    .welcome-section li {
        flex-direction: column;
        align-items: center;
    }

    br {
        display: none;
    }
}
```
Getting this:
image


What do you think???

**Review by Mariii-23 — CHANGES_REQUESTED** [review]

File: src/pages/Home/styles.css (hunk 1 above)
Comment (Mariii-23) [reply src/pages/Home/styles.css:52]: we can also remove the ```br```, getting this in pc mode
image

Comment (assebc) [reply src/pages/Home/styles.css:52]: seems good to me, i'll change that in a sec

File: src/pages/Home/index.tsx [hunk 2]
```diff
@@ -1,52 +1,102 @@
...
+
+        <div className="meeting-section">
+            <Layout.Body>
+                <Layout.Center>
+                    <h1>It seems you're ready to take the next step</h1>
+                    <p>Talking with us it's free and we will probably give precious tips.</p>
+                    <button onClick={() => handleNavigate(navigate, "/contact-us")}>Schedule a Meeting</button>
+                </Layout.Center>
+            </Layout.Body>
+        </div>
```
Comment (Mariii-23) [inline src/pages/Home/index.tsx:97]: Create a molecule component, so that we can use it in other pages

**Review by Mariii-23 — CHANGES_REQUESTED** [review]

**Review by Mariii-23 — APPROVED** [review]

**Pull request merged**